vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -208,6 +209,13 @@ private VCFHeader addMissingMetadataLines(VCFHeader header) {
         if (header.getFormatHeaderLine("GT") == null) {
             header.addMetaDataLine(new VCFFormatHeaderLine("GT", 1, VCFHeaderLineType.String, "Genotype"));
         }
+        // CSQ line
+        if (header.getFormatHeaderLine("CSQ") == null) {
+            header.addMetaDataLine(new VCFInfoHeaderLine("CSQ", 1, VCFHeaderLineType.String,
```

Comment:
The annotation is not going to be added always (in VariantExporterController there is an option to exclude the annotation). If we are excluding the annotation, this fields should not be added to the Vcf header.